### PR TITLE
Minor fix

### PR DESCRIPTION
**Pull request by dreinon**

### CODE REVIEW

**Comment by tpaviot** [comment]
Great, did you use a linter?

File: processscheduler/problem.py [hunk 1]
```diff
@@ -214,8 +217,8 @@ def add_objective_flowtime_single_resource(self, resource, time_interval=None, w
         # for this resource, we look for the minimal starting time of scheduled tasks
         # as well as the maximum
         flowtime = Int('FlowtimeSingleResource%s_%s' % (resource.name, uid))
-        
-        flowtime_single_resource_indicator = Indicator('FlowTime(%s:%i:%s)' % (resource.name,
+
+        flowtime_single_resource_indicator = Indicator('FlowTime(%s:%i:%i)' % (resource.name,
```
Comment (dreinon) [inline processscheduler/problem.py:221]: This is wrong. It should stay as it was

**Comment by dreinon** [comment]
> Great, did you use a linter?

Yes, I did automatically. I try not to modify style if I have to change just a bit of code, but if I modify enough code, linter + formatter help me with readability and error preventing.

**Comment by dreinon** [comment]
I use vscode as editor, with flake8 linting (this doesn't format code, just reports errors in code and in style) and autopep8 formatter (this does modify code). Both format my code style as PEP guide states (or at least, almost :smile:)
Vscode python extension has itself a linter called Pylance, which is also very good, and which I also use.

In addition, I set some precommit hooks to achieve what formatter doesn't. Between all I use, I think the following would be interesting for this project:
    -   id: trailing-whitespace (remove whitespaces from end of lines)
    -   id: end-of-file-fixer (adds one line to the end of file if none)
    -   id: check-yaml (checks yaml files format is correct)
    -   id: check-added-large-files (checks that no >10000 kb files are commited)
        args: [--maxkb=10000]
    -   id: double-quote-string-fixer (changes double for single quotes)
    -   id: mixed-line-ending (detects mixed line endings like eof, newline, etc)
        args: [--fix=lf]
    -   id: check-ast (checks no import * are used)
    -   id: check-merge-conflict (what name says)
    -   id: autopep8 (applies autopep8 formatting if not applied)

If you want, review my PR whenever you have time and after closing them, I can make a PR with my code style proposal.

**Comment by tpaviot** [comment]
This is a too big commit only for a small fix. I suggest splitting the fix and the coding style changes into two different PR. It will be easier to review

**Comment by dreinon** [comment]
Ok, I agree this PR is not so accurate. I'll review it later, close this and open a new one.

**Pull request closed without merging**